### PR TITLE
version 0.1.2 - Food ID and key

### PR DESCRIPTION
**Pull request by ichbestimmtnicht**
The lib is not compatible with the latest food API - fixed

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -6,7 +6,9 @@
 e = Edaman(nutrition_appid='5a32xxx',
            nutrition_appkey='cabexxx',
            recipes_appid='ecxxxb',
-           recipes_appkey='83347a87xxxde8106646')
+           recipes_appkey='83347a87xxxde8106646',
+           food_appid='ecxxxb',
+           food_appkey'83347a87xxxde8106646')
```
Comment (ichbestimmtnicht) [inline example.py]: I forgot an = -> New request.

**Review by ichbestimmtnicht — COMMENTED** [review]
I am reopening the Request. I am sorry for the mistake.

File: example.py (hunk 1 above)
Comment (ichbestimmtnicht) [reply example.py]: Fixed

File: example.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
            recipes_appid='ecxxxb',
            recipes_appkey='83347a87xxxde8106646',
            food_appid='ecxxxb',
-           food_appkey'83347a87xxxde8106646')
+           food_appkey='83347a87xxxde8106646')
```
Comment (ichbestimmtnicht) [inline example.py:11]: Fixed

**Comment by JarbasAl** [comment]
looks good, will test later today, thank you!

**Pull request merged**